meta-balena-common/recipes-support/hostapp-update-hooks/files/99-resin-grub

```diff
@@ -61,17 +61,20 @@ fi
 grub_cfg=$(find $RESIN_BOOT_MOUNTPOINT -name grub.cfg)
 grub_env=$(find $RESIN_BOOT_MOUNTPOINT -name grubenv)
 
-if  grep -q upgrade_available "$grub_cfg" ; then
+if  grep -q upgrade_available "$grub_env" ; then
```

Comment:
On long/medium term we want to unify bootloader configuration (uboot and grub) in such a way that they can both read the same data. When that is done, we will stop sed-ing grub_cfg. Let's handle that separately and leave the current setup for now.

Reply:
Dropped for now. Made https://github.com/balena-os/meta-balena/issues/1583 to track this